infrastructure/kube/keep-test/geth-node/eth-goerli-node.yaml

```diff
@@ -44,7 +44,7 @@ spec:
     spec:
       containers:
         - name: geth-goerli
-          image: ethereum/client-go:v1.10.20
+          image: ethereum/client-go:latest
```

Comment:
I'm not sure about using the `latest` tag here. For example, Geth 1.10.22 had a bug that was fixed later in Geth 1.10.23. We should be aware of the version we use and ensure the version is stable before upgrading it.